sycl/test-e2e/Basic/span.cpp

```diff
@@ -3,6 +3,7 @@
 //
 // Fails to release USM pointer on HIP for NVIDIA
 // XFAIL: hip_nvidia
+// XFAIL-TRACKER: https://github.com/intel/llvm/issues/14404
```

Comment:
Is this correct? The issue is about AMD HIP, not NVidia.

Reply:
Thanks for the catch, fixed it!

Reply:
I don't see how this issue is useful though. If the HW is really unsupported, it should be a `REQUIRES: !arch-<something>` or something like that.

Or, going deeper, I don't see a value in just adding a bunch of formal links if they don't become actionable.

\+ @AlexeySachkov

Reply:
I tend to agree with Andrei here. If we don't support this configuration (and AFAIK, we don't), then there is no reason for us to maintain those `XFAIL`s, we can just drop them.